### PR TITLE
Refactor public page of event to make use on new API for existing fields

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Refactor public page of event to make use on new API for existing fields

#### Changes proposed in this pull request:
Update the components which were using old API to fetch any kind of data
- image 
- date 
- URL
image

### CODE REVIEW

File: app/templates/public.hbs [hunk 1]
```diff
@@ -34,7 +34,7 @@
         {{outlet}}
       </div>
       <div class="three wide column" style="height: 100vh; overflow: hidden; !important">
-        {{public/social-links eventUrl=model.eventUrl socialLinks=model.socialLinks}}
+        {{public/social-links eventUrl=model.url socialLinks=model.socialLinks}}
```
Comment (niranjan94) [inline app/templates/public.hbs]: Have you included social links in the modal call ?

File: app/routes/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ const { Route } = Ember;
 export default Route.extend({
   model() {
     return this.store.query('event', {
-      include : 'event-topic,event-sub-topic,event-type,speakers-call',
+      include : 'event-topic,event-sub-topic,event-type,speakers-call,social-links',
```
Comment (niranjan94) [inline app/routes/index.js]: Not here. For the public events page. It isn't needed here.

File: app/templates/public.hbs (hunk 1 above)
Comment (CosmicCoder96) [inline app/templates/public.hbs]: @niranjan94 done

File: app/routes/public.js [hunk 3]
```diff
@@ -9,6 +9,6 @@ export default Route.extend({
   },
 
   model(params) {
-    return this.store.findRecord('event', params.event_id);
+    return this.store.findRecord('event', params.event_id, {include: 'social-links'});
```
Comment (niranjan94) [inline app/routes/public.js]: add a space after `{` and before `}`

Comment (CosmicCoder96) [inline app/routes/public.js]: done 👍

**Pull request merged**